boards/BMS/Src/App/App_Accumulator.c

```diff
@@ -169,10 +198,17 @@ void App_Accumulator_RunOnTick100Hz(struct Accumulator *const accumulator)
 
             UPDATE_PEC15_ERROR_COUNT(accumulator->read_cell_voltages(), accumulator->num_comm_tries)
 
-            // Write to configuration register to configure cell discharging
-            // TODO: re-enable
-            // accumulator->write_cfg_registers();
-            accumulator->disable_discharge();
+            if (accumulator->is_cell_balancing_enabled)
```

Comment:
I feel like enabling balancing should be done during run on entry in the balancing state. Same for disabling balancing, should be done in run on exit